src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.command.Command;
+
+/**
+ * A Personal Assistant Chatbot that helps a person to keep track of various things.
+ * CS2103T iP
+ * AY22/23 Semester 1
+ *
+ * @author Perry Wong
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
```

Comment:
I like the way you name the collection attribute here!

Reply:
Thanks!